src/js/player.js

```diff
@@ -3521,7 +3521,17 @@ class Player extends Component {
   resetProgressBar_() {
     this.currentTime(0);
```

Comment:
Do we really need to do this since the call to `el.load()` will reset everything ?
https://github.com/videojs/video.js/blob/2a99a7824ea175836249a4044c1d530cce174e08/src/js/tech/html5.js#L1356-L1382

Reply:
It might not be necessary to call `this.currentTime(0)` but I'm not 100% sure about that so I'd be happy still keeping that in the `resetProgressBar_` fn